src/main/java/uk/gov/register/presentation/HtmlViewSupport.java

```diff
@@ -28,6 +28,10 @@ public static String fieldLink(String fieldName) {
         return UriBuilder.fromPath("http://field.openregister.org/field").path(fieldName).build().toString();
     }
 
+    public static String publicBodyLink(String publicBodyId){
+        return UriBuilder.fromPath("http://public-body.openregister.org/public-body").path(publicBodyId).build().toString();
```

Comment:
does this url have to be hardcoded? can't we use the link rendering code we use elsewhere for field links?